profile-widget/diveeventitem.cpp

```diff
@@ -186,6 +186,7 @@ void DiveEventItem::setupToolTipString(struct gasmix *lastgasmix)
 			name += QString::fromUtf8(mb.buffer, mb.len);
 		}
 		*lastgasmix = *mix;
+		free_buffer(&mb);
```

Comment:
this is a nitpick.
i would reduce the scope of the `free_buffer(&mb);` here and place it bellow the `name += QString::fromUtf8(mb.buffer, mb.len);` line. it is relevant only if the `put_format()` call above succeeds.